test/test_codec.py

```diff
@@ -5,10 +5,13 @@
 import unittest
 import tempfile
 import os
-from canmatrix import formats
+import decimal
 
+from canmatrix import formats
 from canmatrix.canmatrix import Signal
 
+D = decimal.Decimal
```

Comment:
We can see what @ebroecker says but I'm not a fan of this.  It makes it basically impossible to search for or see uses of the thing at a glance.

Reply:
I am that lazy :-( I'll get rid of it.